cumulusci/tasks/bulkdata/step.py

```diff
@@ -478,7 +482,7 @@ def _convert(rec):
         }[self.operation]
 
         for chunk in iterate_in_chunks(
-            self.api_options.get("batch_size", 200), records
+            self.api_options.get("batch_size") or DEFAULT_REST_BATCH_SIZE, records
```

Comment:
The REST API supports a max batch size of 200. We're validating for that in the mapping parser when the API is explicitly set to REST, but not when it's set to Smart. In that case we could end up here with too high a batch size and need to clamp it.

Reply:
Fixed in e94780b7